ngi_pipeline/engines/piper_ngi/local_process_tracking.py

```diff
@@ -358,14 +362,24 @@ def update_coverage_for_sample_seqruns(project_id, sample_id, piper_qc_dir,
     for libprep_id, seqruns in seqruns_by_libprep.iteritems():
         for seqrun_id in seqruns:
             label = "{}/{}/{}/{}".format(project_id, sample_id, libprep_id, seqrun_id)
+            genome_results_file_paths=glob.glob(os.path.join(piper_qc_dir, "{}.{}*.qc".format(sample_id, seqrun_id.split('_')[-1]),"genome_results.txt"))
             ma_coverage = parse_mean_coverage_from_qualimap(piper_qc_dir, sample_id, seqrun_id)
+
+            reads=0
+            for path in genome_results_file_paths:
+                try:
+                    reads += parse_qualimap_reads(genome_results_file_path)
+                except IOError as e :
+                    LOG.error("Cannot find the genome_results.txt file to get the number of reads in {}".format(genome_results_file_path))
```

Comment:
And that's why my rule is to never deploy anything on fridays afternoons :x

On Fri, Jun 3, 2016 at 1:07 PM, Pontus Larsson <notifications@github.com>
wrote:

> In ngi_pipeline/engines/piper_ngi/local_process_tracking.py
> <https://github.com/NationalGenomicsInfrastructure/ngi_pipeline/pull/339#discussion_r65691307>
> :
>
> >              ma_coverage = parse_mean_coverage_from_qualimap(piper_qc_dir, sample_id, seqrun_id)
> > +
> > +            reads=0
> > +            for path in genome_results_file_paths:
> > +                try:
> > +                    reads += parse_qualimap_reads(genome_results_file_path)
> > +                except IOError as e :
> > +                    LOG.error("Cannot find the genome_results.txt file to get the number of reads in {}".format(genome_results_file_path))
>
> .format(path)
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/NationalGenomicsInfrastructure/ngi_pipeline/pull/339/files/c4fb49d5a512ae52a40825c754e9274df9b15b63#r65691307>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AHLL0LAe0Dc9CNcdOvTBPaYZi4ZpsZGFks5qIArzgaJpZM4IVbMJ>
> .
>